cmd/uncloud/service/stop.go

```diff
@@ -0,0 +1,64 @@
+//nolint:dupl
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/compose/v2/pkg/progress"
+	"github.com/docker/docker/api/types/container"
+	"github.com/psviderski/uncloud/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+type stopOptions struct {
+	services       []string
+	signal         string
+	timeoutChanged bool
+	timeout        int
+}
+
+func NewStopCommand() *cobra.Command {
+	opts := stopOptions{}
+	cmd := &cobra.Command{
+		Use:   "stop SERVICE [SERVICE...]",
+		Short: "Stop one or more services.",
+		Long:  "Stop one or more services.",
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			uncli := cmd.Context().Value("cli").(*cli.CLI)
+			opts.services = args
+			opts.timeoutChanged = cmd.Flags().Changed("timeout")
+			return stop(cmd.Context(), uncli, opts)
+		},
+	}
+	cmd.Flags().StringVarP(&opts.signal, "signal", "s", "", "Signal to send to the container")
+	cmd.Flags().IntVarP(&opts.timeout, "timeout", "t", 0, "Seconds to wait before killing the container")
```

Comment:
Note: I kept the "default" `0` here as that's what the docker code does: https://github.com/docker/cli/blob/master/cli/command/container/stop.go#L48